src/compute/src/bin/computed.rs

```diff
@@ -11,22 +11,21 @@ use std::net::SocketAddr;
 use std::path::PathBuf;
 use std::process;
 
-use anyhow::{bail, Context};
+use anyhow::bail;
 use axum::routing;
-use mz_compute::server::CommunicationConfig;
-use mz_service::secrets::SecretsReaderCliArgs;
 use once_cell::sync::Lazy;
 use tracing::info;
 
 use mz_build_info::{build_info, BuildInfo};
+use mz_compute::server::CommunicationConfig;
 use mz_compute_client::service::proto_compute_server::ProtoComputeServer;
 use mz_orchestrator_tracing::TracingCliArgs;
 use mz_ore::cli::{self, CliConfig};
 use mz_ore::metrics::MetricsRegistry;
 use mz_ore::now::SYSTEM_TIME;
 use mz_pid_file::PidFile;
 use mz_service::grpc::GrpcServer;
-use mz_storage::types::connections::ConnectionContext;
+use mz_service::secrets::SecretsReaderCliArgs;
```

Comment:
Stray?

Reply:
Nope, just a fix of import order. That import was in the third-party import block before.

Reply:
We still need the `--secret-*` command line arguments, because our orchestrator unconditionally passes them to both storaged and computed and clap isn't happy about unknown arguments. Rather than messing with the orchestrator, I figured we can just leave the secret arguments in place, as we might need them at some point in the future.